src/modules/esl-carousel/plugin/autoplay/esl-carousel.autoplay.mixin.ts

```diff
@@ -1,29 +1,33 @@
 import {ExportNs} from '../../../esl-utils/environment/export-ns';
-import {attr, bind, listen, ready} from '../../../esl-utils/decorators';
+import {bind, listen, ready} from '../../../esl-utils/decorators';
 
 import {ESLCarouselPlugin} from '../esl-carousel.plugin';
 import {ESLCarouselSlideEvent} from '../../core/esl-carousel.events';
 
+export interface ESLCarouselAutoplayConfig {
+  /** Timeout to send next command to the host carousel */
```

Comment:
Is it really timeout? Or delay?

Reply:
It sounds like we need to discuss variable name as well.
I'll add a note to the cleanup ticket. Let's revisit it separately